dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN opam init -a -y -j $NJOBS --compiler="$COMPILER" default https://opam.ocaml.
 # Common OPAM packages.
 # `num` does not have a version number as the right version to install varies
 # with the compiler version.
-ENV BASE_OPAM="num ocamlfind.1.8.0 jbuilder.1.0+beta20 ounit.2.0.8" \
+ENV BASE_OPAM="num ocamlfind.1.8.0 dune.1.0.0 ounit.2.0.8" \
     CI_OPAM="menhir.20180530 elpi.1.0.4 ocamlgraph.1.8.8"
```

Comment:
@ejgallego : can you explain the consequences of the jbuilder vs dune change? Is this related to #6857 ? Isn't in this PR intentionally?

Reply:
They are the same software, `jbuilder` got renamed to `dune` due to trademark issues.